styles.css

```diff
@@ -1 +1,82 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.container {
+    background-color: pink;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Hé dit had ik ook xD Je mag dit ook als repeat (3, 1fr) schrijven al is dat in dit geval niet veel korter dan 3 maal 1fr. Maar bij grotere aantallen colums zal dat wel makkelijker zijn!